node-fs-readfile/cat.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const [, , ...rest] = process.argv;
+
+let counter = 0;
+const arr = [];
+
```

Comment:
The array is actually redundant if you go the recursion route. Just print the data from the current readfile before continuing to the next one.